hackathon_site/dashboard/frontend/src/api/helpers.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    Order,
+    OrderInTable,
+    OrderItemTableRow,
+    ReturnedItem,
+    ReturnOrderInTable,
+} from "./types";
+
+export const teamOrderListSerialization = (
```

Comment:
So the teamOrderListSerizalization essentially categorizes the orders?

Reply:
yes it serializes the backend data into a format that's usable on the frontend (in this case, the data we want to see in the order tables)